src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colori getRandomColor() {
```

Comment:
According to the checklist, it's not good practice to create multiple instances of the same class when one can be reused. In this case, you're creating a new instance of `Random` every time `getRandomColor()` is called. It would be better to create a single instance of `Random` as a class-level variable and use it in your method.